### DISCUSSION TITLE
Deleted/Archived themes. What should we do?

### DISCUSSION THREAD

**Discussion by Neikon** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#49] [Q&A]
I hadn't thought about this possibility before and I think it will happen a lot now with proton. What do we do with old themes that either have been archived by their owners or we know they don't work well?

It has just happened with a theme that its owner has removed it from our website and archived its repository. At the same time he has sent us a new theme as a replacement.

Things that come to my mind about this topic:
1. Delete these themes and let them be lost in oblivion. (it would be the easiest but the one I would like the least).
2. Create a markdown file where to list the themes in a very simple way and reference it in the web as we do with "submit your theme".
3. Create a section in the current web to show the old topics as well as the current ones.

Comment what you think about it. @BeyondMagic @andreasgrafen

**Comment by unseen-ninja** [comment]
I personally think that discontinued themes don't need to be listed somewhere as they'll most likely break sooner than later anyway. Especially when the author has archived the repo. Thus I'd opt for the first option. 

That said: Probably the second option is the best. This would still allow users to look at old themes maybe for inspiration or something while it's relatively low maintenance. c:

**Reply by BeyondMagic** [reply]
Sorry y'all for the late response, I agree with andreas, a section showing archived themes would be nice. I wouldn't like to never seeing them any more.

**Comment by Neikon** [comment]
ok, so I will create a list in markdown and link it with a button at the top of the web as we do with "submit themes", etc. that is the easiest way to maintain this.

**Reply by BeyondMagic** [reply]
alright :)
